README.md

```diff
@@ -9,12 +9,79 @@
 
 <img src="assets/spectrum-css_illustration_desktop.png">
 
+## Quick start
+
+### Node/yarn
```

Comment:
I moved this section over from `site/get-started.pug` and did some minor rewording.